pkg/input/wordlist.go

```diff
@@ -91,11 +91,13 @@ func (w *WordlistInput) readFile(path string) error {
 
 	var data [][]byte
 	reader := bufio.NewScanner(file)
+	re := regexp.MustCompile(`(?i)%ext%`)
```

Comment:
For readability, this should get moved to the `if w.config.DirSearchCompat`... block

Reply:
I think outside the loop is better so the regex doesn't get created for every line in the input

Reply:
Absolutely, thanks for clarification.